modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts

```diff
@@ -52,14 +67,27 @@ describe('browser.tinymce.themes.silver.editor.color.ColorPickerSanityTest', ()
       const setHexWhite = setHex('ffffff');
       const setHexBlack = setHex('000000');
 
-      const pSubmitDialog = async (editor: Editor) => {
+      const submit = async (editor: Editor) => {
         const docBody = getBody(editor);
         FocusTools.setFocus(docBody, dialogSelector);
         await Waiter.pTryUntil('Button is not disabled', () => UiFinder.notExists(docBody, 'button.tox-button:contains("Save")[disabled]'));
```

Comment:
Is this even applicable anymore? as the button is never disabled and was likely added when "Ok" once upon a time got disabled (eg during the 5.0 dev preview).